.gitignore

```diff
@@ -0,0 +1,5 @@
+.terraform/
+/.idea/
+.pytest_cache/
+/venv/
+.terraform.lock.hcl
```

Comment:
end of line is missing. I use a vscode configuration which adds a newline at the end  of file if missing.